package.json

```diff
@@ -56,5 +66,17 @@
     "singleQuote": true,
     "trailingComma": "all",
     "printWidth": 80
+  },
+  "staticFiles": {
```

Comment:
Copies the `staticPath` folder contents to the `dist/` folder depending on which environment you are building for.